Lib/fontTools/merge.py

```diff
@@ -993,20 +1078,39 @@ def merge(self, fontfiles):
 			A :class:`fontTools.ttLib.TTFont` object. Call the ``save`` method on
 			this to write it out to an OTF file.
 		"""
-		mega = ttLib.TTFont()
+		# Take first input file sfntVersion
+		sfntVersion = ttLib.TTFont(fontfiles[0]).sfntVersion
```

Comment:
how about you directly sniff the sfntVersion by reading the first 4 bytes, instead of creating a TTFont then throwing it away

Reply:
It does not even ha ve to do this as all fonts are opened a few lines below anyway.